lib/src/model.dart

```diff
@@ -4054,6 +4056,109 @@ abstract class ModelElement extends Canonicalization
     }
   }
 
+  /// Replace &#123;@youtube ...&#125; in API comments with some HTML to embed
```

Comment:
What is "`&#123;`"?

Reply:
Oh is that `{`? hm...
Does it work (not get interpreted as magic) if you put `{` in backticks?

Reply:
Open curly brace.  It needs that so that dartdoc doesn't try to interpret the dartdoc directives in its own docs.